webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js

```diff
@@ -29,6 +29,20 @@ const ActionsBar = ({ hostId, permissions: { edit_hosts: canEdit } }) => {
       {__('Build')}
     </DropdownItem>,
     <DropdownSeparator key="separator" />,
+    <DropdownItem
+      href={`hosts/${hostId}`}
+      key="prev-version"
+      component="button"
+    >
+      {__('Previous version')}
+    </DropdownItem>,
+    <DropdownItem
+      href="https://community.theforeman.org/t/foreman-3-0-new-host-detail-page-feedback/25281"
```

Comment:
This just renders a `<button` with an `href` attribute, so the link isn't working